operator/.bingo/go.mod

```diff
@@ -1 +1,3 @@
-module _ // Fake go.mod auto-created by 'bingo' for go -moddir compatibility with non-Go projects. Commit this file, together with other .mod files.
\ No newline at end of file
+module _ // Fake go.mod auto-created by 'bingo' for go -moddir compatibility with non-Go projects. Commit this file, together with other .mod files.
+
+go 1.21
```

Comment:
(nit) Perhaps we shouldn't change operator files.

Reply:
I can revert, this was picked up through one of the build commands

Reply:
Reverted